### PR TITLE
Use View setter in RVInput.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: model/src/main/scala/explore/optics/GetAdjust.scala [hunk 1]
```diff
@@ -16,6 +16,6 @@ object GetAdjust {
   def apply[T, A](lens: Lens[T, A]): GetAdjust[T, A] =
     GetAdjust(lens.asGetter, lens.asAdjuster)
 
-  def apply[T, A](get: T => A, set: (A => A) => T => T): GetAdjust[T, A] =
-    GetAdjust(Getter(get), Adjuster(set))
+  def apply[T, A](get: T => A, mod: (A => A) => T => T): GetAdjust[T, A] =
+    GetAdjust(Getter(get), Adjuster(mod))
```
Comment (rpiaggio) [inline model/src/main/scala/explore/optics/GetAdjust.scala:20]: This is an unrelated naming fix.

File: targeteditor/src/main/scala/explore/targeteditor/RVInput.scala [hunk 2]
```diff
@@ -87,29 +98,21 @@ object RVInput {
             FormInputEV(
               id = state.rvView.tag,
               label = state.rvView.tag,
-              value = props.value.zoom(unsafeRVtoZLens),
+              value = props.value.zoom(rvToRedshiftGet)(rvToRedshiftMod),
```
Comment (rpiaggio) [inline targeteditor/src/main/scala/explore/targeteditor/RVInput.scala:101]: In cases where we don't have a `Lens` available, it's OK to zoom via get/mod functions.

Also, we allow setting to be done via `InputFormEV` calling the `View`'s `set` rather than doing it manually `onBlur`.

Comment (cquiroz) [reply targeteditor/src/main/scala/explore/targeteditor/RVInput.scala:101]: I like this, I was thinking to do it as the lens are unlawful.

**Review by cquiroz — APPROVED** [review]
LGTM

**Review by toddburnside — APPROVED** [review]
LGTM
